.circleci/config.yml

```diff
@@ -48,7 +48,10 @@ commands:
     steps:
     - checkout
     - run: git fetch --all --tags
-    - run: export COMMIT_MESSAGE=$(git log -1 --pretty=format:%s)
+    - run:
+        command: |
+          export COMMIT_MESSAGE=$(git log -1 --pretty=format:%s)
```

Comment:
i dont think the COMMIT_MESSAGE is used, so think we can remove. Is the COMMIT_MESSAGE_ENV needed ? did the check env vars set by default though circle ci ?

Reply:
- Removed the COMMIT_MESSAGE as it was not used
- Also removed the export of COMMIT_MESSAGE_ENV as it could be replace by the git log... statement in the if then else where it was used. It was not necessary to add it to the pre_step